public/js/render.js

```diff
@@ -12,6 +12,10 @@ var renderModule = function (error, apiData) {
   });
 
   function renderEvents (eventsArray, language) {
+    document.getElementById('arLang').classList.remove('active-lang');
+    document.getElementById('enLang').classList.remove('active-lang');
+    document.getElementById(language + 'Lang').classList.add('active-lang');
```

Comment:
also on the arabic filter button there is a plus. so what does the active language mean ? since you are saying english + arabic, then both of them need to be highlighted you know what i mean?